Source/WebGPU/WebGPU/Device.h

```diff
@@ -76,6 +80,7 @@ class Device : public WGPUDeviceImpl, public ThreadSafeRefCountedAndCanMakeThrea
     Ref<CommandEncoder> createCommandEncoder(const WGPUCommandEncoderDescriptor&);
     Ref<ComputePipeline> createComputePipeline(const WGPUComputePipelineDescriptor&);
     void createComputePipelineAsync(const WGPUComputePipelineDescriptor&, CompletionHandler<void(WGPUCreatePipelineAsyncStatus, Ref<ComputePipeline>&&, String&& message)>&& callback);
+    Ref<ExternalTexture> createExternalTexture(const WGPUExternalTextureDescriptor&);
```

Comment:
That is true however it creates an `ExternalTexture`, so I think we should deviate from the JS specification since it doesn't align with the internal implementation in this case.